README.md

```diff
@@ -1,29 +1,45 @@
 # Framework
 Common framework for all cp3-llbb analyses
 
-## Setup instructions
+*Please note:*
+* The instructions are for the UCLouvain ingrid SLC6 cluster (to access SAMADhi)
+* You need the proper username and password to access SAMADhi :) If you don't know what this is about, ask around
+* The current state of the art mini-AOD documentation can be found [here](https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookMiniAOD2015)
 
-```bash
 
+
+## First time setup instructions
+
+```bash
+source /nfs/soft/grid/ui_sl6/setup/grid-env.sh
+source /cvmfs/cms.cern.ch/cmsset_default.sh
 export SCRAM_ARCH=slc6_amd64_gcc491
-cmsrel CMSSW_7_4_4
-cd CMSSW_7_4_4/src
+cmsrel CMSSW_7_4_5
+cd CMSSW_7_4_5/src
 cmsenv
 
 git cms-init
 cd ${CMSSW_BASE}/src 
 
+# Electron ID as from [June 28th 2015 EGamma hypernews](https://hypernews.cern.ch/HyperNews/CMS/get/egamma/1589.html)
+git cms-merge-topic ikrav:egm_id_74X_v2
```

Comment:
Oh sorry, I missed the MVA thing. It seems only the cut based ones are merged into the release, not the MVA ...